### DISCUSSION TITLE
Getting Pydantic version conflict when installing outlines[serve]

### DISCUSSION THREAD

**Discussion by Quang-elec44** [outlines-dev/outlines#498] [Q&A] [answered]
When I installed `outlines[serve]`, there was a conflict that `vllm==0.2.6` depends on `pydantic==1.10.13` while your lib requires `pydantic>=2.0`. Therefore I couldn't install your lib. Can you help me out?

**Comment by rlouf** [comment]
For now you can reinstall `pydantic>=2.0` after installing vLLM and it should work.

**Reply by Quang-elec44** [reply]
Unfortunately, it doesn't work. Have a look at my terminal when I install your lib. It keeps trying to find a suitable version of vllm
![image](https://github.com/outlines-dev/outlines/assets/51706966/c5a93fdf-2f07-442c-9bc0-0af7e6ae75b3)

**Reply by rlouf** [reply]
And

```
pip install outlines
pip install vLLM
pip install pydantic>=2.0
```

**Comment by rlouf** [comment, marked as answer]
This is tracked in https://github.com/outlines-dev/outlines/issues/504
